### DISCUSSION TITLE
Consent to Send Failure using examples on Edge (Chrome)

### DISCUSSION THREAD

**Discussion by JoshuaHintze** [aiortc/aiortc#1235] [Q&A]
I'am consistently getting Consent to Send Failures after the video has been streaming over webrtc for 10-20 seconds on Edge (Chrome).  Regular Chrome and Safari (iPhone) work great.

### Steps to reproduce 

1. Download or clone the examples directory.
2. Utilize the webcam example.
3. Execute `pip install aiortc aiohttp`.
4. Download a sample MP4 file, such as Big Buck Bunny.
5. Start the webserver using `python3 webcam.py --play-from ./big_buck_bunny_480p_30mb.mp4`.
6. Open a web browser and navigate to localhost:8080.
7. Choose to select or not select the Use Stun checkbox (either option results in an error).
8. After about 10 seconds, the following errors occur, terminating the connection.

```
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.1.11', 46058) -> ('192.168.1.220', 57644)) State.FROZEN -> State.WAITING
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.122.1', 40392) -> ('192.168.1.220', 57644)) State.FROZEN -> State.WAITING
INFO:aioice.ice:Connection(3) Check CandidatePair(('172.17.0.1', 59447) -> ('192.168.1.220', 57644)) State.FROZEN -> State.WAITING
INFO:aioice.ice:Connection(3) Check CandidatePair(('172.18.0.1', 54686) -> ('192.168.1.220', 57644)) State.FROZEN -> State.WAITING
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.1.11', 46058) -> ('38.78.242.228', 57644)) State.FROZEN -> State.WAITING
Connection state is connecting
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.1.11', 46058) -> ('192.168.1.220', 57644)) State.WAITING -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.122.1', 40392) -> ('192.168.1.220', 57644)) State.WAITING -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('172.17.0.1', 59447) -> ('192.168.1.220', 57644)) State.WAITING -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('172.18.0.1', 54686) -> ('192.168.1.220', 57644)) State.WAITING -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.1.11', 46058) -> ('38.78.242.228', 57644)) State.WAITING -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.122.1', 40392) -> ('38.78.242.228', 57644)) State.FROZEN -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('172.17.0.1', 59447) -> ('38.78.242.228', 57644)) State.FROZEN -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('172.18.0.1', 54686) -> ('38.78.242.228', 57644)) State.FROZEN -> State.IN_PROGRESS
INFO:aioice.ice:Connection(3) Check CandidatePair(('192.168.1.11', 46058) -> ('192.168.1.220', 57644)) State.IN_PROGRESS -> State.SUCCEEDED
INFO:aioice.ice:Connection(3) ICE completed
Connection state is connected
INFO:aioice.ice:Connection(3) Consent to send expired
Connection state is closed
```

Ive tried this on different computers, different linux devices (WSL or Native Ubuntu). 

I have noted that this example for webcam with playing back a file does not work in Firefox with or without STUN but that was already noted in the README.md for the example.

I'm mostly reporting this here for anyone else who has this issue with Edge (Chrome based) browsers.

**Comment by gavrie01** [comment]
Have faced the same problem a few times. Would be interesting to know if there is fair explanation / solution.
Thanks in advance!

**Reply by Victorious-hub** [reply]
Any updates? Faced the same exception, creating my own chat app with aiortc. On firefox works fine, but Chrome, microsoft edge calls Connection(0) Consent to send expired after 30 seconds

**Comment by JoshuaHintze** [comment]
I moved onto mediaMTX project. Its a great software solution for video streaming.
